vpr/src/device/rr_graph_view.h

```diff
@@ -259,12 +259,12 @@ class RRGraphView {
         return node_storage_.node_side_string(node);
     }
 
-    /** @brief Get the number of configurable edges. This function is inlined for runtime optimization. */
+    /** @brief Get the number of configurable edges, each configurable edge is controlled by a programmable memory bit. This function is inlined for runtime optimization. */
```

Comment:
Can you enrich the code comments by following the reference:

https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/36cc78fe8ed5eb223e56899b6f67662b0aba2bf0/vpr/src/device/rr_graph_obj.h#L26-L31

We may enrich the comment like 
```
A configurable edge represents a programmable switch between routing resources, which could be 
- a multiplexer
- a tri-state buffer
- a pass gate

Reply:
The current comment looks pretty good, but the enrichment you suggest would be good to add as well.